src/lib/Component/Registry.php

```diff
@@ -8,48 +8,137 @@
 
 namespace Ibexa\AdminUi\Component;
 
-use Ibexa\Contracts\AdminUi\Component\Renderable;
+use Ibexa\Contracts\TwigComponents\ComponentInterface;
+use Ibexa\TwigComponents\Component\Registry as TwigComponentsRegistry;
 
+/**
+ * @deprecated 4.6.19 The {@see \Ibexa\AdminUi\Component\Registry} class is deprecated, will be removed in 5.0.
+ * Use {@see \Ibexa\TwigComponents\Component\Registry} instead
+ */
 class Registry
 {
-    /** @var \Ibexa\Contracts\AdminUi\Component\Renderable[] */
-    protected $components;
-
     /**
-     * @param \Ibexa\Contracts\AdminUi\Component\Renderable[] $components
+     * @var string[]
      */
-    public function __construct(array $components = [])
+    private const COMPONENTS_WHITELIST =
```

Comment:
this prolly should be compiler pass with appending new values, but given that there is minor gain from this, and this is already deprecated :+1: